src/scss/components/_list.scss

```diff
@@ -84,13 +92,11 @@
     display: flex;
     flex-shrink: 0;
     font-size: 14px;
-    margin-bottom: 2px;
-    margin-top: 2px;
+    margin-bottom: 8px;
```

Comment:
The good practice is to have always a `margin-top`. And I **recommend** to use the gorgeous `get-space`. 😉

Reply:
It will not work that margin-top approach because of our fluid list. I'll add the `get-space`.

Reply:
Ok, make sense. Go ahead! 👍